code/io/exportYaml.m

```diff
@@ -248,9 +248,9 @@ function writeMetadata(model,fid)
 fprintf(fid, ['    full_name: "', model.description, '"\n']);
 fprintf(fid, ['    version: "', model.version, '"\n']);
 fprintf(fid, ['    date: "', datestr(now,29), '"\n']);  % 29=YYYY-MM-DD
-fprintf(fid, ['    authors: "', model.annotation.authorList, '"\n']);
-fprintf(fid, ['    email: "', model.annotation.email, '"\n']);
-fprintf(fid, ['    organization: "', model.annotation.organization, '"\n']);
+%fprintf(fid, ['    authors: "', model.annotation.authorList, '"\n']);
+%fprintf(fid, ['    email: "', model.annotation.email, '"\n']);
+%fprintf(fid, ['    organization: "', model.annotation.organization, '"\n']);
```

Comment:
This is easy code to write, so for maintainability my suggestion is to remove it completely:
```suggestion
```